Formula/bison.rb

```diff
@@ -1,9 +1,11 @@
 class Bison < Formula
   desc "Parser generator"
   homepage "https://www.gnu.org/software/bison/"
-  url "https://ftp.gnu.org/gnu/bison/bison-3.5.91.tar.xz"
-  mirror "https://ftpmirror.gnu.org/bison/bison-3.5.91.tar.xz"
-  sha256 "9145f59d184c5703a3857c074c1a570c9606be0e9471dd6548c9192e80ce30da"
+  # X.Y.9Z are beta releases that sometimes get accidentally uploaded to the release FTP
```

Comment:
Can we add a livecheck rule/audit for this?

Reply:
Probably. Livecheck usually handles this stuff (though not in this case).

For an audit, if it can be generalised to something like all GNU software then maybe.

Reply:
livecheck does handle this. (maybe we can include livecheck as part of workflow??)

Reply:
This is the commit ref in the livecheck side: https://github.com/samford/homebrew-livecheck/commit/972ff29c742ce2770d62413a6ac6bc3898f74c80